exercises/practice/gilded-rose/gilded_rose_test.rb

```diff
@@ -0,0 +1,274 @@
+  end
+
+  def test_normal_item_of_zero_quality
+    update_with sell_in: 1, quality: 0, name: "some item", quality_change: 0
+  end
+
+  def test_normal_item_near_zero_quality
+    update_with sell_in: 0, quality: 1, name: "some item", quality_change: -1
+  end
+
+  def test_brie_before_sell_date
+    update_with sell_in: 1, quality: 0, name: "Aged Brie", quality_change: 1
+  end
+
+  def test_brie_on_sell_date
+    update_with sell_in: 0, quality: 0, name: "Aged Brie", quality_change: 2
+  end
+
+  def test_brie_after_sell_date
+    update_with sell_in: -1, quality: 0, name: "Aged Brie", quality_change: 2
+  end
+
+  def test_brie_of_max_quality
+    update_with sell_in: 1, quality: 50, name: "Aged Brie", quality_change: 0
+  end
+
+  def test_brie_near_max_quality
+    update_with sell_in: 0, quality: 49, name: "Aged Brie", quality_change: 1
+  end
+
+  def test_sulfuras_before_sell_date
+    update_with sell_in: 1, quality: 80, name: "Sulfuras, Hand of Ragnaros",
+      sell_in_change: 0, quality_change: 0
+  end
+
+  def test_sulfuras_on_sell_date
+    update_with sell_in: 0, quality: 80, name: "Sulfuras, Hand of Ragnaros",
+      sell_in_change: 0, quality_change: 0
+  end
+
+  def test_sulfuras_after_sell_date
+    update_with sell_in: -1, quality: 80, name: "Sulfuras, Hand of Ragnaros",
+      sell_in_change: 0, quality_change: 0
+  end
+
+  def test_backstage_pass_far_from_sell_date
+    update_with sell_in: 11, quality: 10, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 1
+  end
+
+  def test_backstage_pass_medium_from_sell_date_upper_bound
+    update_with sell_in: 10, quality: 0, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 2
+  end
+
+  def test_backstage_pass_medium_from_sell_date_lower_bound
+    update_with sell_in: 6, quality: 0, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 2
+  end
+
+  def test_backstage_pass_close_to_sell_date_upper_bound
+    update_with sell_in: 5, quality: 0, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 3
+  end
+
+  def test_backstage_pass_close_to_sell_date_lower_bound
+    update_with sell_in: 1, quality: 0, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 3
+  end
+
+  def test_backstage_pass_on_sell_date
+    update_with sell_in: 0, quality: 10, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: -10
+  end
+
+  def test_backstage_pass_after_sell_date
+    update_with sell_in: -1, quality: 10, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: -10
+  end
+
+  def test_backstage_pass_of_max_quality
+    update_with sell_in: 11, quality: 50, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 0
+  end
+
+  def test_backstage_pass_near_max_quality
+    update_with sell_in: 1, quality: 49, name: "Backstage passes to a TAFKAL80ETC concert",
+      quality_change: 1
+  end
```

Comment:
These tests are all missing the `skip` part

Reply:
These tests cover the existing features in the starting code, so they are passing in the beginning. Should they still be skipped?

I checked in a track where refactoring exercises are implemented, and I found that sometimes they are skipped ([1](https://github.com/exercism/csharp/blob/main/exercises/practice/tree-building/TreeBuildingTests.cs), [2](https://github.com/exercism/csharp/blob/main/exercises/practice/ledger/LedgerTests.cs)), but not always ([3](https://github.com/exercism/csharp/blob/main/exercises/practice/markdown/MarkdownTests.cs)).

Reply:
Well, maybe they shouldn't. But then they should _all_ be unskipped

Reply:
Done!